note.txt

```diff
@@ -0,0 +1,64 @@
+switch branch: git checkout fix/kelvin-setup
+
+conda env:
+
+(base) C:\Users\User>conda info --envs
+# conda environments:
+#
+base                  *  C:\Users\User\miniconda3
+
+(base) C:\Users\User>conda create -n condaEnv1
+Retrieving notices: ...working... done
+Channels:
+ - defaults
+Platform: win-64
+Collecting package metadata (repodata.json): done
+Solving environment: done
+
+#
+# To activate this environment, use
+#
+#     $ conda activate condaEnv1
+#
+# To deactivate an active environment, use
+#
+#     $ conda deactivate
+
+
+(base) C:\Users\User>conda info --envs
+# conda environments:
+#
+base                  *  C:\Users\User\miniconda3
+condaEnv1                C:\Users\User\miniconda3\envs\condaEnv1
+
+
+install nodejs:
```

Comment:
Once you install `npm` and `node.js`, you can simply just run `npm i` under the `client` folder, this command will pull all the libraries needed by this project.